tests/test_config.py

```diff
@@ -169,6 +169,22 @@ def test_getboolean_error(self):
         with self.assertRaises(ValueError):
             self.cli_config.getboolean(section, option)
 
+    def test_items_case_insensitive(self):
+        section = 'MYSECTION'
+        envOption = '{}_{}_{}'.format(CLIConfig._DEFAULT_CONFIG_ENV_VAR_PREFIX, section, 'TEST_OPTION')
+        envVaule = 'envValue'
+        configOption = 'test_option'
+        configValue = 'value'
```

Comment:
The python convention is to use snake case.

Reply:
> 
> 
> The python convention is to use snake case.

Agree